src/main/resources/mybatis/collections.xml

```diff
@@ -79,34 +79,34 @@
     <select id="getCollectionsJson" parameterType="map" resultType="String">
         select json_build_object(
             'collections', json_agg(collections),
-             'links' , json_build_array(
-                 json_build_object(
-                      'type', 'application/json',
-                       'rel',  'self',
-                       'title', 'This document as JSON',
-                       'href',  #{serverUrl} || 'collections?f=json'
-                 )
-             )
+            'links' , json_build_array(
+                json_build_object(
+                    'type', 'application/json',
+                    'rel',  'self',
+                    'title', 'This document as JSON',
+                    'href',  #{serverUrl} || 'collections?f=json'
+                )
+            )
```

Comment:
I tried to auto-format my code and got a lot of auto-formatting indent changes.  It all works and is a bit more consistent now in indenting.

Reply:
yuck - https://teams.microsoft.com/l/entity/com.microsoft.teamspace.tab.wiki/tab::b84f615b-4850-4f11-843e-f723d7b856c6?context=%7B%22subEntityId%22%3A%22%7B%5C%22pageId%5C%22%3A36%2C%5C%22sectionId%5C%22%3A39%2C%5C%22origin%5C%22%3A2%7D%22%2C%22channelId%22%3A%2219%3A5e685c105f234374937c119669a8c294%40thread.skype%22%7D&tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494